modules/dmrpp_module/vlsa_util.cc

```diff
@@ -340,7 +330,8 @@ void read(const pugi::xml_node &vlsa_element, std::vector<std::string> &entries)
         }
         BESDEBUG(VLSA, prolog << "value: '" << value << "' count: " << count << "\n");
         for(uint64_t i=0; i<count ;i++){
-            entries.emplace_back(value);
+            // Using emplace_back(value) here causes an error (why?) with ASAN on. jhrg 2/11/25
```

Comment:
the semantics of emplace_back are confusing and I think there are issues with using it that we should better understand!!

Reply:
True. I'm not completely sure why ASAN flags this particular line. The issue is a (detected) write beyond the end of the container, but then ASAN says, (paraphrasing) "if you want to ignore these errors, define this ENV variable." So is this important or not? In this case, I switched to push_back() because the other uses of emplace_back() work fine - this was the only one that produced this warning/error from ASAN.